nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/S3ColdStorageFeature.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (C) Copyright 2021 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Abdoul BA<aba@nuxeo.com>
+ */
+
+package org.nuxeo.coldstorage;
+
+import com.amazonaws.auth.AWSCredentials;
+import org.nuxeo.ecm.core.test.CoreFeature;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+import static org.nuxeo.ecm.blob.s3.S3BlobStoreConfiguration.*;
+import static org.nuxeo.ecm.blob.s3.S3BlobStoreConfiguration.SYSTEM_PROPERTY_PREFIX;
+
+@Features({ S3ColdStorageFeature.SetPropertiesFeature.class, CoreFeature.class })
+public class S3ColdStorageFeature extends  ColdStorageFeature{
```

Comment:
I think we can add `@Deploy("org.nuxeo.coldstorage.test:OSGI-INF/test-s3-coldstorage-contrib.xml")` on the feature itself, here in the case of S3.